src/main/java/org/reaktivity/nukleus/socks/internal/SocksConfiguration.java

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright 2016-2017 The Reaktivity Project
+ *
+ * The Reaktivity Project licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package org.reaktivity.nukleus.socks.internal;
+
+import org.reaktivity.nukleus.Configuration;
+
+public class SocksConfiguration extends Configuration
+{
+    public static final String SOCKS_INITIAL_WINDOW = "nukleus.socks.initial.window";
```

Comment:
I am not sure this applies here, because there could be the case of 2 nuklei working together while fine-tuning the values independently would then not be possible. Currently, the initial window is the total number of bytes the handshake takes (the 2 messages). I was planning to send back a window after receiving each message during handshake. In this case, the option would be something like `nukleus.socks.max.handshake.message` and could be as low as 261 bytes, until implementing other authentication methods or extensions which might require larger messages.

Reply:
Since as we discussed yesterday we have a firm basis for knowing 300 will always be sufficient, let's just make it a constant and get rid of the SocksConfiguration class. We could call it for example SOCKS_MAXIMUM_MESSAGE_SIZE.